include/sdf/Geometry.hh

```diff
@@ -141,6 +145,17 @@ namespace sdf
     /// \param[in] _cylinder The cylinder shape.
     public: void SetCylinderShape(const Cylinder &_cylinder);
 
+    /// \brief Get the ellipsoid geometry, or nullptr if the contained
+    /// geometry is not an ellipsoid.
+    /// \return Pointer to the visual's ellipsoid geometry, or nullptr if the
```

Comment:
I know that this is probably repeating the pattern above, but I'll just point out that the geometry may be attached to a collision, not necessarily a visual.

Reply:
Adjusted wording in this comment